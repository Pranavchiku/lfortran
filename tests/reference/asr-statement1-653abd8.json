{
    "basename": "asr-statement1-653abd8",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/statement1.f90",
    "infile_hash": "3d828667fda659d080d10043c790e05aa1fa73d1f920b2c4e38fe751",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-statement1-653abd8.stdout",
    "stdout_hash": "c4e4ae056a693e2bdefff953cbac24797188131aa7e305381d47d68a",
    "stderr": "asr-statement1-653abd8.stderr",
    "stderr_hash": "744c35cc0d7b383bd2284c683c8155af8bcbc8551fd0c4f768c0965f",
    "returncode": 0
}