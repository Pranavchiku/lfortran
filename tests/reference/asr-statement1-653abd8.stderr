warning: Implied save feature
 --> tests/statement1.f90:2:2
  |
2 |  double precision x, a, b, c
  |  ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/statement1.f90:2:2
  |
2 |  double precision x, a, b, c
  |  ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/statement1.f90:2:2
  |
2 |  double precision x, a, b, c
  |  ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/statement1.f90:2:2
  |
2 |  double precision x, a, b, c
  |  ^^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables

warning: Implied save feature
 --> tests/statement1.f90:3:2
  |
3 |  logical q
  |  ^^^^^^^^^ Assuming save attribute implicitly for variables
